packages/lesswrong/components/posts/PostsPage/PostsPageDate.jsx

```diff
@@ -35,6 +35,19 @@ const PostsPageDate = ({ post, hasMajorRevision, classes }) => {
       Curated on <ExpandedDate date={post.curatedDate}/>
     </div>}
   </div>);
+
+  if (hasMajorRevision) {
+    return (
+      <span>
+        <span className={classNames(classes.date, classes.mobileDate)}>
```

Comment:
Should this be using the tooltip?

Reply:
No, because it has its own tooltip. All the changes to this file are because we displayed two tooltips on top of each other.